app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,69 @@
+class Admin::UsersController < ApplicationController
+    before_action :set_user, only: [:show, :edit, :update, :destroy]
+    #before_action :require_login
+
+    def index
+      @users = User.all
+      unless current_user.admin?
+        redirect_to user_path(current_user), notice: '管理者権限ページへアクセスは出来ません!'
+      end
+    end
+
+    def create
+        @user = User.new(user_params)
+        if @user.save
+          redirect_to user_path(@user.id)
+        else
+          render :new
+        end
+    end
+
+    def new
+        @user = User.new
+    end
+
+    def edit
+    end
+
+    def show
+      @user = User.find(params[:id])
+      @tasks = @user.tasks
+    
+    end
+
+    def update
+      @user.update(user_params)
+      if User.where(admin: :true).count == 0
```

Comment:
0件チェックは下記の様にも記載できます。
```suggestion
      if User.where(admin: :true).zero?
```